front-end-master/landing_page/index.html

```diff
@@ -171,33 +170,55 @@ <h1 class="headline">Our Associates</h1>
     </div>
 
 
-
-	<div class="form-container scrollify">
-		<div class="wrapper">
-            <div class="form-content">
-                <div class="form-headline-container">
-                    <h1 class="form-headline">GET IN TOUCH</h1>
-                    <h3 class="form-headline-sub">Let us know your thoughts</h3>
-                </div>
-                <form class="form-body">
-                    <div class="form-name-container form-group">
-                        <input type="text" class="form-control" placeholder="First Name">
-                        <input type="text" class="form-control" placeholder="First Name">
+    <form id="contact-us" class="container contact-card-container" action="https://formspree.io/thersquaredproject@gmail.com" method="POST">
+        <div class="row">
+          <div class="col-sm">
+            <div class="card">
+                <div class="card-body">
+                  <div class="contact-heading">
+                      <span>GET IN TOUCH</span>
+                      <span>Let us know your thoughts</span>
+                  </div>
+                  <div class="contact-texboxes">
+                    <div class="form-row">
+                        <div class="col-6">
+                          <input type="text" class="form-control" name="First Name" required="required" placeholder="First name">
+                        </div>
+                        <div class="col-6">
+                          <input type="text" class="form-control" name="Last Name" required="required" placeholder="Last name">
+                        </div>
                     </div>
-                    <div class="form-mail-container form-group">
-                        <input type="email" class="form-control" placeholder="Example@gmail.com">
+                    <div class="form-row">
+                        <div class="col-12">
+                            <input type="email" class="form-control" name="Email" required="required" placeholder="Email">
+                        </div>
                     </div>
-                    <div class="form-message-container form-group">
-                        <textarea name="message" placeholder="Your Message" class="form-control"></textarea>
+                    <div class="form-row">
+                        <div class="col-12">
+                            <textarea class="form-control" rows="3" name="Message" required="required" placeholder="Write a message.."></textarea>
+                        </div>
                     </div>
-                    <button class="btn">Submit</button>
-                </form>
-            </div>
-
-            <div class="form-sidebar">
+                    <div class="form-row" style="margin-top: 20px">
```

Comment:
since it was a one-time thing I needed to do so I just decided to write it inline instead of overwriting the "form-row" class and having the changes everywhere.